projects/opendr_ws/src/perception/scripts/object_tracking_2d_siamrpn.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
```

Comment:
Could you please rename the node script to `object_tracking_2d_siamrpn_node.py`, as all the ros1 nodes will be renamed this way in #364. That PR will probably be merged before this, and the package name will also change from `perception` to `opendr_perception`.

Other than that the node looks good!